docs/.gitignore

```diff
@@ -0,0 +1,20 @@
+# Dependencies
+/node_modules
+
+# Production
+/build
+
+# Generated files
+.docusaurus
+.cache-loader
+
+# Misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
```

Comment:
Can we use something like *.log

Reply:
this is coming as default `.gitignore` from docusaurus. Anyway, will fix it.